decode/render.ts

```diff
@@ -251,14 +254,14 @@ export function resize(data: ResizeData, iframe: HTMLIFrameElement): void {
 }
 
 export function change(data: InputChangeData, iframe: HTMLIFrameElement): void {
-    let el = element(data.target) as HTMLInputElement;
+    let el = element(data.target as number) as HTMLInputElement;
```

Comment:
>target [](start = 26, length = 6)

Nit: Why do you need this override? Shouldn't the be in the Typescript?

Reply:
This one's tricky and I've gone back and forth on this. The fact that we have to hold target element as a Node until very late when we are ready to turn it into an id to send over the wire makes it hard. The alternate option would be to fork all interfaces that have target and introduce two fields - one for target Node and another for target Id. I started going down that road and felt we ended up making our interfaces confusing. With this change target can have two possible types (which is defined in data.d.ts) - Node or number. So we cast it in the right format. My preference is what I've in the PR. Let me know if you feel strongly about it and we can chat offline about this. It's probably hard to cover all the issues I encountered over PR.